src/parser/parser.yy

```diff
@@ -538,7 +538,7 @@ unreserved_keyword
     | KW_HTTP               { $$ = new std::string("http"); }
     | KW_HTTPS              { $$ = new std::string("https"); }
     | KW_MERGE              { $$ = new std::string("merge"); }
-    | KW_SPLIT              { $$ = new std::string("split"); }
+    | KW_DIVIDE           { $$ = new std::string("division"); }
```

Comment:
Why change to divide

Reply:
Because SPLIT is a keyword in the function expression